cray_product_catalog/query.py

```diff
@@ -125,7 +125,7 @@ def __init__(self, name=PRODUCT_CATALOG_CONFIG_MAP_NAME, namespace=PRODUCT_CATAL
             str(p) for p in self.products if not p.is_valid
         ]
         if invalid_products:
-            LOGGER.debug(
+            LOGGER.warning(
```

Comment:
I think this was intentionally changed to a debug-level message because there were some products at the time that had data that didn't validate against the schema, and I think it was resulting in a lot of warning messages somewhere (possibly in `sat showrev`) that the admin couldn't really do anything about. I don't know if it will be a problem again or not, just pointing out the rationale. Perhaps it's worth checking on some systems to see if this warning occurs.

Reply:
We set this to warning as our test case which checks for invalid schema was failing with message "self._raiseFailure(
AssertionError: no logs of level DEBUG or higher triggered on root"